### PR TITLE
540 added alias support

### PR DESCRIPTION
**Pull request by mikeerickson**
@philtrep 

I have implemented this is a different way, I think this is pretty clean 

- [x] Created a sensible default `aliases.sh` file in the `workspace` directory
- [x] Modified Dockerfile to copy this file to `/home/laradock/aliases.sh` 
- [x] Updated `~/.bashrc` to source the new file

NOTES:

- User can modify the `aliases.sh` file to customize their aliases

Let me know your thoughts...  If we still with this approach, I will update the `README.md` to contain this additional information

### CODE REVIEW

File: workspace/aliases.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+COL_CYAN=$ESC_SEQ"36;01m"
+
+# Commonly Used Aliases
+alias ..="cd .."
+alias c="clear"
+alias cla="clear && ls -l"
+alias cll="clear && ls -la"
+alias cls="clear && ls"
+alias code="cd /var/www"
+alias e="vi"
```
Comment (philtrep) [inline workspace/aliases.sh]: vim is installed by default for the 'laradock/workspace' image

File: workspace/aliases.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+alias ea="reload"
+alias ea="vi ~/aliases"
+alias ea="vi ~/aliases"
+alias g="gulp"
+alias home="cd ~"
+alias npm-global="npm list -g --depth 0"
+alias reload="source ~/.aliases && echo \"$COL_GREEN ==> Aliases Reloaded... $COL_RESET \n \""
+alias run="npm run"
+alias sshkey="cat ~/.ssh/id_rsa.pub | pbcopy && echo 'Copied to clipboard.'"
```
Comment (philtrep) [inline workspace/aliases.sh]: Public key inside the container?

**Comment by philtrep** [comment]
@mikeerickson Aside from my comments, this looks good. As soon as the changes are done and the doc is updated ill merge this in 👍

**Comment by mikeerickson** [comment]
@philtrep All done, aliases adjust and updated docs

**Comment by philtrep** [comment]
@mikeerickson Thanks, you rock!

File: README.md [hunk 3]
```diff
@@ -410,7 +413,8 @@ You can select your own combination of Containers form the list below:
 ```bash
 docker-compose exec workspace bash
 ```
-Alternatively, for Windows Powershell users: execute the following command to enter any running container:
+
+Alternativey, for Windows Powershell users: execute the following command to enter any running container:
```
Comment (alfaproject) [inline README.md]: typo

**Comment by bagart** [comment]
/home/laradock/aliases.sh
1.
```
/home/laradock/aliases.sh: line 2: $'\r': command not found
```
FIX:
```
sed -i 's/\r//' /home/laradock/aliases.sh
```

2. 
```#! /bin/sh
function mkd() {
    mkdir -p "$@" && cd "$@"
}
```
sh not work firh func!

FIX:
```
sed -i 's/^#! \/bin\/sh/#! \/bin\/bash/' /home/laradock/aliases.sh
```


2sad

**Pull request merged**